packages/common/src/store/pages/audio-rewards/slice.ts

```diff
@@ -277,7 +284,8 @@ export const {
   resetRewardClaimedToast,
   updateOptimisticListenStreak,
   setUserChallengeCurrentStepCount,
```

Comment:
thought just occurred to me - what happens when we set the step count to 1 for 1 step challenges 🤔  (probably ideal to have your action anyway but curious if that works)

Reply:
Good thought but doesn't seem to work for me.